go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/PuerkitoBio/goquery v1.8.1
 	github.com/avast/retry-go v3.0.0+incompatible
 	github.com/aws/aws-sdk-go v1.44.253
-	github.com/aws/karpenter-core v0.27.4-0.20230503190441-611e27dd0aef
```

Comment:
We should be bumping this to the latest HEAD right? Not the latest tagged version?